### PR TITLE
mirror-images-enable-qe-catalogsource

### PR DESCRIPTION
**Pull request by suprs**
Create qe-app-registry by adding step "mirror-images-enable-qe-catalogsource".

@xiaojiey Can you please help to review this?

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-stable.yaml [hunk 1]
```diff
@@ -49,11 +49,10 @@ tests:
...
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
-      E2E_RUN_TAGS: '@aws-ipi and @network-openshiftsdn and not @fips'
-      TAG_VERSION: '@4.11'
+      TEST_SCENRAIOS: Security_and_Compliance
     test:
-    - chain: openshift-e2e-test-qe
-    workflow: cucushift-installer-rehearse-aws-ipi
+    - ref: openshift-extended-test
+    workflow: cucushift-installer-rehearse-aws-ipi-ovn-fips
```
Comment (xiaojiey) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-stable.yaml:55]: currently, the qe-app-registry not available in the profile. You need to enable it first.

Comment (suprs) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-stable.yaml:55]: I have enabled can you review it again, thanks

Comment (xiaojiey) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-stable.yaml:55]: Sorry, the file I pointing to is wrong, should be https://github.com/openshift/release/blob/master/ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/fips/provision/cucushift-installer-rehearse-aws-ipi-ovn-fips-provision-chain.yaml#L7

File: ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/fips/cucushift-installer-rehearse-aws-ipi-ovn-fips-workflow.yaml [hunk 2]
```diff
@@ -5,6 +5,7 @@ workflow:
     - chain: cucushift-installer-rehearse-aws-ipi-ovn-fips-provision
     test:
     - ref: cucushift-installer-check
+    - ref: enable-qe-catalogsource
```
Comment (xiaojiey) [inline ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/fips/cucushift-installer-rehearse-aws-ipi-ovn-fips-workflow.yaml:8]: Please recover it. Thanks.

**Comment by kuiwang02** [comment]
@suprs @xiaojiey 
if you just want to try if your module case is ok to run in prow, could use this PR for trial. but it can not be merged.
if you want to enable your module in prow ci, such PR is not good because you change current test. I will make PR for it.

by the way, qe-app-registry is already enabled in workflow by @jianzhangbjz . you do not need to enable it.

**Comment by kuiwang02** [comment]
/hold

in case it is merged incorrectly

**Pull request closed without merging**